R/GaussSuppressionFromData.R

```diff
@@ -144,12 +145,25 @@
 #'                   Please note that interval calculations may have a 
 #'                   different interface in future versions.
 #'                                 
+#' @param aggregatePackage Package use to preAggregate/extraAggregate. 
```

Comment:
Typo: use -> used

Reply:
Thanks , I will change in final submission to CRAN